From d240fe6a3f98d136b6ba2d315df49e37f7e33636 Mon Sep 17 00:00:00 2001
From: Herbert Xu <herbert@gondor.apana.org.au>
Date: Tue, 6 Jul 2010 17:46:06 +0800
Subject: [EVAL] Removed dead code for eval NPIPE

The notyet code is identical to the current code.

Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
(cherry picked from commit a0212c5dcdd8f2bb69834f2638cddb427dea78a6)

Signed-off-by: Jonathan Nieder <jrnieder@gmail.com>
---
 ChangeLog  |    1 +
 src/eval.c |    6 ------
 2 files changed, 1 insertions(+), 6 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index 149cbc0..8430565 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,6 +1,7 @@
 2010-07-06  Herbert Xu <herbert@gondor.apana.org.au>
 
 	* Fix loss of variables when hash collides.
+	* Removed dead code for eval NPIPE.
 
 2010-06-28  Gerrit Pape <pape@smarden.org>
 
diff --git a/src/eval.c b/src/eval.c
index d142412..58739f5 100644
--- a/src/eval.c
+++ b/src/eval.c
@@ -254,13 +254,7 @@ checkexit:
 		goto calleval;
 	case NPIPE:
 		evalfn = evalpipe;
-#ifdef notyet
-		if (eflag && !(flags & EV_TESTED))
-			checkexit = ~0;
-		goto calleval;
-#else
 		goto checkexit;
-#endif
 	case NCASE:
 		evalfn = evalcase;
 		goto calleval;
-- 
1.7.4.1

